backend/src/lib.rs

```diff
@@ -421,6 +427,62 @@ multicontrolled_qubit_rotation!(
     QuantumSim::mcrz
 );
 
+/// QIR API for applying a joint rotation Pauli-Y rotation with the given angle for the two target qubit.
+#[no_mangle]
+pub extern "C" fn __quantum__qis__rxx__body(
+    theta: c_double,
+    qubit1: *mut c_void,
+    qubit2: *mut c_void,
+) {
+    __quantum__qis__h__body(qubit1);
+
+    __quantum__qis__h__body(qubit2);
+
+    __quantum__qis__rzz__body(theta, qubit1, qubit2);
+
+    __quantum__qis__h__body(qubit2);
+
+    __quantum__qis__h__body(qubit1);
+}
+
+/// QIR API for applying a joint rotation Pauli-Y rotation with the given angle for the two target qubit.
+#[no_mangle]
+pub extern "C" fn __quantum__qis__ryy__body(
+    theta: c_double,
+    qubit1: *mut c_void,
+    qubit2: *mut c_void,
+) {
+    __quantum__qis__h__body(qubit1);
+    __quantum__qis__s__body(qubit1);
+    __quantum__qis__h__body(qubit1);
+
+    __quantum__qis__h__body(qubit2);
+    __quantum__qis__s__body(qubit2);
+    __quantum__qis__h__body(qubit2);
+
+    __quantum__qis__rzz__body(theta, qubit1, qubit2);
+
+    __quantum__qis__h__body(qubit2);
+    __quantum__qis__s__adj(qubit2);
+    __quantum__qis__h__body(qubit2);
+
+    __quantum__qis__h__body(qubit1);
+    __quantum__qis__s__adj(qubit1);
+    __quantum__qis__h__body(qubit1);
+}
+
+/// QIR API for applying a joint rotation Pauli-Z rotation with the given angle for the two target qubit.
+#[no_mangle]
+pub extern "C" fn __quantum__qis__rzz__body(
```

Comment:
The readme covering all of the supported gate operations should be updated with all of these additions.

Reply:
I'll update the readme with new functions and add signatures as well.